keras/utils/generic_utils.py

```diff
@@ -337,7 +337,7 @@ def update(self, current, values=None):
                     self._values[k][0] += v * (current - self._seen_so_far)
                     self._values[k][1] += (current - self._seen_so_far)
             else:
-                self._values[k] = v
+                self._values[k] = [v, 1]
```

Comment:
Why?

Reply:
To distinguish a numerical value (this means taking an average from a single-item list, i.e. just keeping the numerical value the same - I'll add a comment) from a non-numerical log value that should be copied verbatim to output.  This way, the floats are properly formatted the same as other numerical metrics.